_unittest/test_35_MaxwellCircuit.py

```diff
@@ -114,3 +114,11 @@ def test_08_import_netlist(self):
         self.aedtapp.insert_design("SchematicImport")
         self.aedtapp.modeler.schematic.limits_mils = 5000
         assert self.aedtapp.create_schematic_from_netlist(self.netlist_file1)
+
+    def test_09_create_voltage_source(self):
+        id = self.aedtapp.modeler.schematic.create_vsin(name="voltage_source", value=3.14, angle=90)
+        assert id.parameters["Va"] == "3.14"
```

Comment:
@DaveTwyman this property doesn't exist. You create it with the set_property. If you use set_property and input a property name that doesn't exist, this will be created.

Reply:
@gmalinve , Got it id is changed to component in the unit tests